src/Sylius/Behat/Resources/config/suites/api/admin/dashboard.yaml

```
@@ -0,0 +1,37 @@
+# This file is part of the Sylius package.
+# (c) Sylius Sp. z o.o.
+
+default:
+    suites:
+        api_dashboard:
+            contexts:
+                - sylius.behat.context.hook.doctrine_orm
+                    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)